### ISSUE TITLE
Landing Page Improvements - Definition of Open Humans

### ISSUE THREAD

**Issue by jiraslama** [alan-turing-institute/AutisticaCitizenScience#348]
**As a Fujitsu developer**

**I would like**
A wireframe design and justification that showcases a clearer short definition of what Open Humans is and how it relates to the project. 

**Because...**
I would like show to the wider community a Fujitsu design concept for the platform

**Acceptance Criteria**
Axure updated with the final UI design for the component 
A picture/screenshot image of the final UI design of the component held within Axure 
Justification of the component design available in Github repository as an .md file

**Other relevant documents:**
Values 
Ways of working 
Data management

**Review owner:**
Internal development team 
Alan Turing

**User Feedback Labels:**
Landing Page
Open Humans

**Comment by katoss** [comment]
# Original user testing feedback

From https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/user-tests/sprint-6-feedback/summary-sprint6-feedback.md

**‘How it works’ section**

- A C d: “I want to look only at ‘how it works’.”
- A C d: “‘Authorise project’, what does that mean? Is that the consent-thing?”
- A C d: “I would maybe put ‘authorise project’ in brackets and write ‘give consent’, because that describes in plain English what ‘authorise project’ means.”

**Being redirected to the Open Humans platform**

- d: 3x not confusing, 2x confusing
- A D d: Unclear what Open Humans is and why you need to login there
- A D d: “Having to login to Open Humans makes me a bit nervous. I would like more information before signing up and logging in, because you don’t really know what you are logging in for.”
- A (1) d: “I don’t find it confusing. I like the fact that it allows you to use your username or email. I would recognize these, it is positive.”
- A (1) d: “I believe it [that you need to login via Open Humans] was mentioned beforehand to me.”
- A (2) d: “I struggle with the organisations. I mean there is Turing..I don’t really know what Open Humans is. I have heard that phrase many times, I am sure that I have been told, but I don’t know. It is some kind of platform - but I don’t even know what a platform is, I am not a technical person. So that is a problem. But I would think here, if I had already signed up before, I would just look up the login details in my notes. I wouldn’t know what Open Humans is, but I have heard this terminology before.”

**→ Open Humans and why you need to login via this platform should be explained. The fact that the majority of the participants was not confused by the redirection to Open Humans might be explainable by the fact that they had heard about it before in prior contributions to the project.**

**Comment by jiraslama** [comment]
Definition of OH - Added to aims to support login / register process via OH

Partner with Open Humans, adapting their systems for the storing of data, the experience tool will collect. 

Attention to aims - written in a way which acts as a aim and also gives a summary of the purpose of OH usage which the platform. 

https://k6ej5a.axshare.com/#g=1&p=1-landing-page-presentation-tech-expanded-access-o_1&view=b

**Comment by GeorgiaHCA** [comment]
Thanks for doing this @jiraslama - I'm not sure if Open Humans should be in the "aims" section. While they have values which correspond to our goals, partnering with Open Humans isn't an aim of the platform. Defnitely useful feedback though! Clearly Open Humans does need explaining, but in nmy view not in the aims section.
